nservicebus/sqlserver/operations-scripting.md

```diff
@@ -31,6 +31,10 @@ snippet:sqlserver-powershell-nativesend
 
 snippet:sqlserver-nativesend-usage
 
+The `MessageTypeToSend` dictionary value represents the message .NET type, by default the endpoint receiving the message will use that message type to deserialize the incoming message. Depending on the [versioning](/samples/versioning/sample.md) strategy the type can be expressed in 2 different ways:
```

Comment:
i actually dont think this content should be in operations-scripting.md. perhaps somewhere in the serialization doco or the header doco. then link to that from here. @mauroservienti thoughts?